bolt-design-system/bolt
# Epic v2: BDS Homepage Redesign

## Jira

http://vjira2:8080/browse/BDS-507

## Summary

New design for Bolt Design System home page.

## Details

1. New home page implemented
2. Components overview page updated
3. PL pages dropdown organized

![screen shot 2018-08-13 at 3 44 42 pm](https://user-images.githubusercontent.com/3027663/44054143-d6bb3072-9f0f-11e8-957b-acaa9125a23b.png)

## How to test

1. Pull down the branch to your local
2. Run `npm run start` on the `apps/bolt-site` folder
3. Check the landing page
